### ISSUE TITLE
Cannot invoke "com.gargoylesoftware.htmlunit.html.DomElement.click()" because "element" is null

### ISSUE THREAD

**Issue by cmoulliard** [halkyonio/primaza-poc#321]
## Issue

```
2023-05-23 12:29:52,867 WARN  [com.gar.htm.jav.hos.Window] (main) window.confirm("Are you sure ?") no confirm handler installed, simulating the OK button
Error:  Tests run: 7, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.838 s <<< FAILURE! - in io.halkyon.ClaimsEndpointTest
Error:  io.halkyon.ClaimsEndpointTest.testEditClaimFromPage  Time elapsed: 0.116 s  <<< ERROR!
java.lang.NullPointerException: Cannot invoke "com.gargoylesoftware.htmlunit.html.DomElement.click()" because "element" is null
	at io.halkyon.utils.WebPageExtension$PageManager.clickOn(WebPageExtension.java:143)
	at io.halkyon.utils.WebPageExtension$PageManager.clickById(WebPageExtension.java:92)
	at io.halkyon.ClaimsEndpointTest.testEditClaimFromPage(ClaimsEndpointTest.java:95)
```

**Comment by Sgitario** [comment]
It seems that this test is no longer valid from: https://github.com/halkyonio/primaza-poc/commit/71a5317ba7ecbc5a9ae251e4f32fee05ead18734

**Comment by aureamunoz** [comment]
fixed by #324
